ROADMAP.md

```diff
@@ -2,60 +2,47 @@
 
-More detail about short-term delivery plans can be found in the [milestones](https://github.com/vmware-tanzu/community-edition/milestones) in
-the [GitHub project repo](https://github.com/vmware-tanzu/community-edition).
+This roadmap should serve as a reference point for Tanzu Community Edition users to help us prioritize existing features and provide input on unmet needs, and for contributors to understand where the project is heading. A roadmap also helps our contributor community to determine if a proposed enhancement conflicts with the long-term plan.
 
-## How to help
+Enhancement states help everyone understand if something is in the early stages of an idea or if it has wider community buy-in and committed resources. This is important because it helps limit waste, and focus work on the most important and timely activities. The stages are described below:
 
-Discussion on the roadmap can take place in threads under [Issues](https://github.com/vmware-tanzu/community-edition/issues)
-or in [community meetings, our Google Group, or our Slack channel](https://tanzucommunityedition.io/community/).
-Please open and comment on an issue if you want to provide suggestions and feedback to an item in the roadmap.
-Please review the roadmap to avoid potential duplicated effort.
+.Awaiting Proposal - these enhancements are in the idea mode, and the project maintainers are awaiting a detailed proposal. This state is useful to explore different solutions to a known problem and encourage discussion.
+.Proposal in Review - maintainers are currently reviewing the proposal, which includes resource requirements and commitments.
+.Implementation in Progress - engineering work has begun and will be tracking towards a release target.
 
-## Need an idea for a contribution?
+## How to help
 
-Check out our [issues](https://github.com/vmware-tanzu/community-edition/issues) for issues tagged
-[help-wanted](https://github.com/vmware-tanzu/community-edition/issues?q=is%3Aopen+is%3Aissue+label%3Ahelp-wanted) or
-[good-first-issue](https://github.com/vmware-tanzu/community-edition/labels/good-first-issue).
+Discussion on the roadmap can take place in threads under Issues or in community meetings, our Google Group, or our Slack channel. Please open and comment on an issue if you want to provide suggestions for feedback on  an item in the roadmap.  Community members are encouraged to be actively involved, and also stay informed so contributions can be made with the most positive effect and limited duplication of effort.
 
 ## How to add an item to the roadmap
 
-Please open an issue to track any initiative on the roadmap (usually driven by new feature requests). We will work with
-and rely on our community to focus our efforts to improve Tanzu Community Edition.
+Please open an issue to track any initiative on the roadmap (usually driven by new feature requests). We will work with and rely on our community to focus our efforts to improve Tanzu Community Edition.
```

Comment:
I don't understand this. Is it about "requesting a new item to be on the roadmap"? Like the title says?\

It says: "open an issue **to track any initiative on the roadmap**" <-- implies it pre-exists, right?